docs/message.md

```diff
@@ -17,54 +17,73 @@ More precisely:
 
 ## How to use
 The central component for the Message API is the `MessageService`.
-It exposes all the services mentioned above and is accessible from the `SymphonyBdk` object by calling the `messages()` method.
-For instance:
-
+It exposes all the services mentioned above and is accessible from the `SymphonyBdk` object by calling the `messages()` method:
 ```java
-import com.symphony.bdk.core.MessageService;
-
 public class Example {
   public static final String STREAM_ID = "gXFV8vN37dNqjojYS_y2wX___o2KxfmUdA";
 
   public static void main(String[] args) throws Exception {
     // Create BDK entry point
     final SymphonyBdk bdk = new SymphonyBdk(loadFromClasspath("/config.yaml"));
-    // Get the MessageService object
-    final MessageService messageService = bdk.message();
-
-    //send a regular message
-    final V4Message regularMessage = messageService.send(STREAM_ID, "<messageML>Hello, World!</messageML>");
-    System.out.println("Message sent, id: " + regularMessage.getMessageId());
+    // send a regular message
+    final V4Message regularMessage = bdk.message().send(STREAM_ID, "<messageML>Hello, World!</messageML>");
+    log.info("Message sent, id: " + regularMessage.getMessageId());
   }
 }
 ```
 
-A more detailed example with all exposed services can be found [here](../symphony-bdk-examples/bdk-core-examples/src/main/java/com/symphony/bdk/examples/MessageExampleMain.java).
+## Using templates
+The `MessageService` also allows you to send messages using templates. So far, the BDK supports two different template
+engine implementations: 
+- [FreeMarker](https://freemarker.apache.org/) (through dependency `com.symphony.platformsolutions:symphony-bdk-template-freemarker`)
+- [Handlebars](https://github.com/jknack/handlebars.java) (through dependency `com.symphony.platformsolutions:symphony-bdk-template-handlebars`)
 
-## Send messages with templates
-The Message service also allows you to send messages using templates. So far, we only support [FreeMarker templates](https://freemarker.apache.org/),
-but we may add support for other template engines.
-For instance, if you have the following template file:
+### How to send a message from a template
+> In the code examples below, we will assume that FreeMarker as been selected as template engine implementation
```

Comment:
should we detail how the selection is made (i.e whatever is in the classpath is used)?
One could think that there is an API to call to select the engine

Reply:
I documented how to select the implementation. And I also created `APP-3113` to fix the current behaviour which was to randomly decide which implementation is selected by default